CleanSlate/events/wol_business_events.txt

```diff
@@ -505,100 +426,24 @@ character_event = {
 				ai = yes
 				controls_religion = no
 				is_adult = yes
+				is_incapable = no
 				is_within_diplo_range = ROOT
 				mercenary = no
 				holy_order = no
+				prisoner = no
 				primary_title = {
 					is_landless_type_title = no
 				}
 				NOR = {
 					is_rival = ROOT
-					is_incapable = yes
-					prisoner = yes
-					religion_group = ROOT
-					ROOT = {
-						any_liege = {
-							character = PREVPREV
-						}
-					}
+					is_liege_or_above_of = ROOT
 				}
 			}
-			character_event = { id = WoL.10113 }
-			break = yes
-		}
-		random_independent_ruler = {
-			limit = {
-				ai = yes
-				controls_religion = no
-				is_adult = yes
-				is_within_diplo_range = ROOT
-				mercenary = no
-				holy_order = no
-				primary_title = {
-					is_landless_type_title = no
-				}
-				NOR = {
-					is_rival = ROOT
-					is_incapable = yes
-					prisoner = yes
-					religion = ROOT
-					ROOT = {
-						any_liege = {
-							character = PREVPREV
-						}
-					}
-				}
+			preferred_limit = {
+				NOT = { religion_group = ROOT }
 			}
-			character_event = { id = WoL.10113 }
-			break = yes
-		}
-		random_independent_ruler = {
-			limit = {
-				ai = yes
-				controls_religion = no
-				is_adult = yes
-				is_within_diplo_range = ROOT
-				mercenary = no
-				holy_order = no
-				primary_title = {
-					is_landless_type_title = no
-				}
-				NOR = {
-					is_rival = ROOT
-					is_incapable = yes
-					prisoner = yes
-					ROOT = {
-						any_liege = {
-							character = PREVPREV
-						}
-					}
-				}
-			}
-			character_event = { id = WoL.10113 }
-			break = yes
-		}
-		random_independent_ruler = {
-			limit = {
-				ai = yes
-				controls_religion = no
-				is_adult = yes
-				mercenary = no
-				holy_order = no
-				primary_title = {
-					is_landless_type_title = no
-				}
-				NOT = {
-					OR = {
-						is_rival = ROOT
-						is_incapable = yes
-						prisoner = yes
-						ROOT = {
-							any_liege = {
-								character = PREVPREV
-							}
-						}
-					}
-				}
+			preferred_limit = {
+				NOT = { religion = ROOT }
```

Comment:
`			preferred_limit = {
				NOT = { religion_group = ROOT }
				is_within_diplo_range = ROOT
			}
			preferred_limit = {
				NOT = { religion = ROOT }
				is_within_diplo_range = ROOT
			}
			preferred_limit = {
				NOT = { religion_group = ROOT }
			}
			preferred_limit = {
				NOT = { religion = ROOT }
			}`